gensim/models/wrappers/wordrank.py

```diff
@@ -126,7 +125,7 @@ def train(cls, wr_path, corpus_file, out_name, size=100, window=15, symmetric=1,
         with smart_open(cooccurrence_shuf_file, 'rb') as f:
             numlines = sum(1 for line in f)
         with smart_open(meta_file, 'wb') as f:
-            meta_info = "{0} {1}\n{2} {3}\n{4} {5}".format(numwords, numwords, numlines, cooccurrence_shuf_file, numwords, vocab_file)
+            meta_info = "{0} {1}\n{2} {3}\n{4} {5}".format(numwords, numwords, numlines, cooccurrence_shuf_file.split('/')[-1], numwords, vocab_file.split('/')[-1])
```

Comment:
Dtto on `split`.

Elsewhere in the file (and in gensim) the standard C-style `%s %d %f` string formatting is used; best to keep it consistent here as well.